_source/android/01_getting_started.md

```diff
@@ -23,7 +23,7 @@ Then select API 28 or higher for the minimum SDK and Kotlin DSL for the build co
 
 ## Integrate Hotwire Native
 
-Add the Hotwire Native dependencies to your app's module (not top-level) `build.gradle.kts` file:
+Add the Hotwire Native dependencies to your app's module (not top-level) `build.gradle.kts` file. You can look up the version number from `https://github.com/orgs/hotwired/packages`.
```

Comment:
Good idea, thanks. Can you turn this into a proper markdown link and link to the android repo releases instead:

```
You can look up the version number from [github.com/hotwired/hotwire-native-android/releases](https://github.com/hotwired/hotwire-native-android/releases).
```